src/object/vertical_stripes.cpp

```diff
@@ -0,0 +1,69 @@
+//  SuperTux
+//  Copyright (C) 2020 Grzegorz Przybylski <zwatotem@gmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "object/vertical_stripes.hpp"
+
+#include "editor/editor.hpp"
+#include "math/rect.hpp"
+#include "math/rectf.hpp"
+#include "supertux/sector.hpp"
+#include "video/drawing_context.hpp"
+#include "video/surface.hpp"
+
+VerticalStripes::VerticalStripes() :
+  m_visible(false),
+  m_layer(LAYER_FOREGROUND1 + 10),
+  m_left_stripe(0,0,0,0),
+  m_right_stripe(0,0,0,0)
+{
+}
+
+VerticalStripes::~VerticalStripes()
+{
+}
+
+void
+VerticalStripes::update(float dt_sec)
+{
```

Comment:
::update() gets run once every frame. I'm wondering whether this should be placed somewhere more appropriate.